test/suite/test_timestamp03.py

```diff
@@ -215,27 +270,51 @@ def test_timestamp03(self):
             count += 1
         # print "Updated " + str(count) + " keys to value2"
 
+        # Scenario : 4
+        # Now check that we don't see the updated data of timestamp tables
+        # with old read_timestamp
+        self.check(self.session, 'read_timestamp=' + timestamp_str(100),
+            self.tablename, dict((k, self.value) for k in orig_keys))
+        self.check(self.session, 'read_timestamp=' + timestamp_str(100),
+            self.tablename2, dict((k, self.value2) for k in orig_keys))
+        self.check(self.session, 'read_timestamp=' + timestamp_str(100),
+            self.tablename3, dict((k, self.value) for k in orig_keys))
+        self.check(self.session, 'read_timestamp=' + timestamp_str(100),
+            self.tablename4, dict((k, self.value2) for k in orig_keys))
+
+        # Scenario : 5
+        # Now check that we see the expected state when reading at each
+        # updated timestamp
+        valdict = dict((k, self.value) for k in orig_keys)
+        for i, t in enumerate(orig_keys):
+            valdict[i+1] = self.value2
+            self.check(self.session, 'read_timestamp=' + timestamp_str(t + 100),
+                self.tablename, valdict)
+            self.check(self.session, 'read_timestamp=' + timestamp_str(t + 100),
+                self.tablename2, dict((k, self.value2) for k in orig_keys))
+            self.check(self.session, 'read_timestamp=' + timestamp_str(t + 100),
+                self.tablename3, valdict)
+            self.check(self.session, 'read_timestamp=' + timestamp_str(t + 100),
+                self.tablename4, dict((k, self.value2) for k in orig_keys))
+
         # Take a checkpoint using the given configuration.  Then verify
         # whether value2 appears in a copy of that data or not.
-        valcnt2 = valcnt3 = nkeys
+        valcnt2 = valcnt3 = valcnt4 = nkeys
```

Comment:
These variables should be renamed to go along with the table and uri renaming to help keep track of what they represent.